tools/roslaunch/src/roslaunch/remoteprocess.py

```diff
@@ -201,6 +204,26 @@ def _ssh_exec(self, command, address, port, username=None, password=None):
                 if str(e).startswith("Unknown server"):
                     pass
                 err_msg = "Unable to establish ssh connection to [%s%s:%s]: %s"%(username_str, address, port, e)
+                err_msg +="""
+
+The fingerprint for the key sent by the remote host is different from the key stored in the known_hosts.
```

Comment:
How is it ensured that the error case is actually the one described here? Couldn't there be a different causing the same exception?